a_players.inc

```diff
@@ -1545,7 +1545,7 @@ native SetPVarString(playerid, const varname[], const string_value[]);
 /// <seealso name="DeletePVar"/>
 /// <remarks>If length of string is zero (value not set), string_return text will not be updated or set to anything and will remain with old data, neccesying that you clear the variable to blank value if <a href="#GetPVarString">GetPVarString</a> returns <b><c>0</c></b> if that behavior is undesired </remarks>
 /// <returns>The length of the string.</returns>
-native GetPVarString(playerid, const varname[], string_return[], len);
+native GetPVarString(playerid, const varname[], string_return[], len = string_return);
```

Comment:
Seems that you missed `sizeof` on this line.

Reply:
thanks, i'll fix that